aws/src/org/purefn/kurosawa/aws/s3.clj

```diff
@@ -46,14 +46,19 @@
 
 ;; temporarily needed while moving from k8s configs where all things are stringly
 ;; typed.
-(def ^:private parse
-  (some-fn #(try (Integer. %)
-                 (catch Exception ex))
-           #(try (Long. %)
-                 (catch Exception ex))
-           #(try (Double. %)
-                 (catch Exception ex))
-           identity))
+(def parse
```

Comment:
Would it make sense to, also, allow for nested json configs, e.g.
```
{"somthing": {"something_else": 1}}
```
I recall trying that in the past, but perhaps it was related to the `str/trim`.

Reply:
It makes sense, but I'd like to ship the bugfix first since what you're asking is a bit more work.